examples/dlint/color.rs

```diff
@@ -1,9 +1,11 @@
 // Copyright 2020-2021 the Deno authors. All rights reserved. MIT license.
-use crate::lexer::{lex, MediaType, TokenOrComment};
-use deno_ast::swc::parser::token::{Token, Word};
+use deno_ast::{
+  lex,
+  swc::parser::token::{Token, Word},
+  MediaType, TokenOrComment,
+};
```

Comment:
I'm not going to fix these using statements in this PR because it's too much work. We should do a format on the files with import granularity = Item once to fix all these.